src/tasks/daily/misc/daily_boat_mixin.py

```
@@ -45,24 +45,25 @@ def collect_clue(self, exchange_help_box):
+            result = self.wait_feature(
+                feature=fL.make_room,
+                time_out=4,
+                box=exchange_help_box,
+                raise_if_not_found=False,
+            )
 
-        if not result:
-            self.mark_task_failure("未找到会客室，任务失败")
-            return False
+            if not result:
+                self.mark_task_failure("未找到会客室，任务失败")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)